backend/lib/services/terminals/terminalBootstrap.js

```diff
@@ -346,16 +332,14 @@ async function handleShoot ({ name, namespace }) {
   Until this is the case we need to workaround this by creating an ingress (e.g. with the respective certmanager annotations) so that a proper certificate is presented for the kube-apiserver.
   https://github.com/gardener/gardener/issues/1413
 */
-async function ensureTrustedCertForShootApiServer (client, shootResource) {
-  const { metadata: { namespace, name } } = shootResource
-  if (!_.isEmpty(shootResource.metadata.deletionTimestamp)) {
+async function ensureTrustedCertForShootApiServer (client, shootResource, seedResource) {
+  const { metadata: { namespace, name, deletionTimestamp } } = shootResource
+  if (!_.isEmpty(deletionTimestamp)) {
     logger.debug(`Shoot ${namespace}/${name} is marked for deletion, bootstrapping aborted`)
     return
   }
 
-  // fetch seed resource
-  const seedName = getSeedNameFromShoot(shootResource)
-  const seedResource = await client['core.gardener.cloud'].seeds.get(seedName)
+  const seedName = seedResource.metadata.name
```

Comment:
Why do we not use the utility function `getSeedNameFromShoot`. If we don't need this supersimple function remove it everywhere.

Reply:
The getSeedNameFromShoot should be used for shoot resources. Here we have the seed resource.